### PR TITLE
update the look of language/locale screen

### PR DESCRIPTION
**Pull request by Muqtxdir**
- changed the layout of welcome screen to better align with [vanilla installer mockup](https://github.com/Vanilla-OS/mockups/issues/2)
- screen shots:

| default state | search state | 
| ------------------- |---------------------- |
| Screenshot from 2023-08-28 10-48-00 | Screenshot from 2023-08-28 10-48-08

_Note: Please dont merge this yet, would be better to merge once keyboard and Timezone page are also adapted_

### CODE REVIEW

**Review by mirkobrombin — APPROVED** [review]

**Review by pietrodicaprio — APPROVED** [review]
Good job!

File: vanilla_installer/gtk/widget-language.ui [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        <property name="justify">center</property>
+        <property name="label" translatable="yes">Language</property>
+        <style>
+          <class name="monospace" />
+          <class name="dim-label" />
+      </style>
+      </object>
+  </child>
+  </template>
+</interface>
```
Comment (kbdharun) [inline vanilla_installer/gtk/widget-language.ui:20]:
```suggestion
</interface>

```

**Review by kbdharun — APPROVED** [review]
The strings LGTM

File: vanilla_installer/gtk/widget-language.ui (hunk 1 above)
Comment (pietrodicaprio) [reply vanilla_installer/gtk/widget-language.ui:20]: The Contribution Guidelines do not include indication of wether the files should end with a new line or not. This suggestion should not be "PR based" but a global rule to be defined if we always want a new line at the end of the files.\
The definition should also state if the files must be LF, CRLF or if doesn't matter

Comment (kbdharun) [reply vanilla_installer/gtk/widget-language.ui:20]: Agreed, we should probably specify that there must be an LF eol in the Contribution Guideline. I will send a PR for it soon. We already set Gitattributes for this in our CLI repos, will do the same for GUI ones.

**Comment by pietrodicaprio** [comment]
@Muqtxdir the issue must not include "feature-update" in the title and the commit must match the issue title.

**Pull request merged**